app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx

```diff
@@ -73,8 +85,14 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
 
     const restartedStage = execution.stages.find(stage => stage.context.restartDetails !== undefined);
 
+    this.displayableParameters = this.getDisplayableParameters(execution);
```

Comment:
there's a prior comment on this line:
https://github.com/armory-io/deck/pull/4#discussion_r270638658
<kbd>![](https://cl.ly/f5e0a4f6df49/download/Screen%20Shot%202019-04-01%20at%2017.37.08.png)</kbd>

Reply:
Hey @kevinawoo id be very happy to get you a mockup. Im crunching on something else at the moment but will make a note to do this later today.